### PR TITLE
[DNM] Bar/Kitchen Overhaul, EPISODE 3: REVENGE OF THE SHIT

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Overview
It's finally here.

https://imgur.com/a/lKm19Xo

### CODE REVIEW

**Comment by BurgerLUA** [comment]
I forgot to delete a backup file. Quickly download it if you want to see what the map looked like several cycles ago.

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 1]
```diff
@@ -64378,7 +64378,7 @@
 /obj/machinery/atm{
 	pixel_x = 5;
 	pixel_y = 26;
-	step_y = 0
+	pixel_y = 0
```
Comment (Arrow768) [inline maps/aurora/aurora-4_mainlevel.dmm]: What are you doing here ?
Defining pixel_y two times ?

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 2]
```diff
@@ -7609,7 +7609,7 @@
 /obj/structure/cable/green,
 /obj/structure/extinguisher_cabinet{
 	pixel_y = -32;
-	step_y = 0
+	pixel_y = 0
```
Comment (Arrow768) [inline maps/aurora/aurora-4_mainlevel.dmm]: And here

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 3]
```diff
@@ -63944,7 +63944,7 @@
 "czB" = (
 /obj/machinery/alarm{
 	pixel_y = 32;
-	step_y = 0;
+	pixel_y = 0;
```
Comment (Arrow768) [inline maps/aurora/aurora-4_mainlevel.dmm]: And here

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 4]
```diff
@@ -7056,6 +7535,118 @@
...
+/obj/machinery/power/apc{
+	dir = 4;
+	name = "east bump";
+	pixel_x = 24;
+	pixel_y = -8
+	},
+/obj/structure/cable/green,
+/obj/structure/extinguisher_cabinet{
+	pixel_y = -32;
+	pixel_y = 0
```
Comment (Arrow768) [inline maps/aurora/aurora-4_mainlevel.dmm]: And here

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 5]
```diff
@@ -64451,292 +63909,1160 @@
...
 	},
-/obj/machinery/door/firedoor,
 /turf/simulated/floor/plating,
-/area/library)
-"cxo" = (
+/area/maintenance/library)
+"czB" = (
+/obj/machinery/alarm{
+	pixel_y = 32;
+	pixel_y = 0;
```
Comment (Arrow768) [inline maps/aurora/aurora-4_mainlevel.dmm]: And here

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 6]
```diff
@@ -64451,292 +63909,1160 @@
...
+	},
+/obj/machinery/camera/network/civilian_east{
+	c_tag = "Bar - North";
+	c_tag_order = 999;
+	dir = 2
+	},
+/obj/machinery/atm{
+	pixel_x = 5;
+	pixel_y = 26;
+	pixel_y = 0
```
Comment (Arrow768) [inline maps/aurora/aurora-4_mainlevel.dmm]: And here

**Comment by BurgerLUA** [comment]
oh no

**Comment by fernerr** [comment]
I know this is work in progress and all, so unsure how helpful this feedback is, but here goes:

There's three emergency shutters missing in these locations. Images for clarity: (https://i.imgur.com/3RIg105.png) (https://i.imgur.com/zonm6tu.png)

The hazard strip decal under the kitchen entrance door is incomplete.

While the disposal mail tags work as planned, the disposal bins within the bar and kitchen don't work as intended, spitting out trash as it can't reach the main disposal tubes.

This you problably are aware of, but just as a reminder neither the main kitchen or bar area are sufficently equipped with air or fire alarms.

The camera network the cameras are set to, Civilian East, Doesn't exist anymore, It was replaced with Civilian Main.

Now some stuff that is just of my opinion:
If the bar shutter is closed, it's a bit dark infront of them, even with nightmode turned off, I suggest putting another light fixture in this position. (https://i.imgur.com/Z874QXG.png)

The cameras should be set to the Service camera network.

Perhaps there should be some obstacle between assistants and them playing on the rafters. A maintenance airlock or a windoor blocking blocking access either up above or on the main level might be good.

Now this is perhaps a bit unrelated, but with the bar usually being a hotspot destination for the wizard's teleport spell from what I've seen, with this addition there's be a decent risk every time they teleport there for them to end up in mid-air and promptly fall down, hurting themselves, as the rafters count as the bar area. Now this is a problem with some other destinations too, but just thought it'd be worth a mention.

**Comment by BurgerLUA** [comment]
I improved the areas around cargo maint as well as the bar and library maint. By areas, I mean the actual "area" object in code. Library maintenance was fucking huge and took up bar maintenance, so I swapped it with bar maintenance.

I also gave the cargo interstitial it's own area instead of it being the library maintenance, even though the library maintenance wasn't connected at all to it.

**Comment by Arrow768** [comment]
My previously raised points have been addressed.

**Comment by LordFowl** [comment]
By moving the freezer to the right side of the kitchen you eliminate the need for the sublevel backroom, which is altogether unnecessary. Removing the vending machines from the corridor further allows you to expand the bar backroom out, and depending on how generous you are with the fast food department.

I don't see the point of the rafters at all. If you really want to include a multi-z dimension to the map, something more suitable would be better, perhaps balcony seating.

Having two minimoogs so close together is going to create a discordance singularity.

I don't know why the theatre maint was moved to the sublevel.

**Comment by BurgerLUA** [comment]
It's done*. Unless travis objects, it's ready to merge.

**Comment by fernerr** [comment]
I like the look of this but here's some things I think need to be adjusted:
The kitchen has two apcs in the same 'area'.
The main kitchen and bar/diner room needs air alarms.

A small idea I had that would fit a professional kitchen: A first aid cabinet, just containing one thing of gauze for eventual knife slipups.

**Comment by BurgerLUA** [comment]
Move this to the september update. Mapmerge fucks up every merge I make and I would appreciate it if it was merged earlier next time so I wouldn't have to deal with mapmerge.

**Pull request closed without merging**